examples/comparison/gorm4/autogenerated_gorm4.go

```diff
@@ -4,6 +4,7 @@ package gorm4
 import (
```

Comment:
Remove `cmd/goqueryset/goqueryset` file plz

Reply:
I don't understand.
This file was there before my commit. I just updated it to be up-to-date.
Isn't it needed for the UTs?

Reply:
Sorry, I didn't understand this file already exists in repo